src/main/java/org/folio/rest/impl/Processor.java

```diff
@@ -175,211 +188,245 @@ private void handleMarcRecordFieldByField(Iterator<DataField> dfIter, Object obj
     }
 
     //there is a mapping associated with this marc field
-    for (int i = 0; i < mappingEntry.size(); i++) {
+    for (int mappingEntryIndex = 0; mappingEntryIndex < mappingEntry.size(); mappingEntryIndex++) {
```

Comment:
Using i as an array index name is fine. It is used once in the next line. Replacing i by a more verbose name does not increase readability.

Reply:
Strongly agree. A longer name connotes something that's going to be around for a while. `i` tells the reader "Ignore this, it's just a loop counter".

Reply:
I partly agree and changed it back at many points. Problems that can arise are nested loops within the same method (indices of which I now called 'ii'). Other point is where the index is compared to another index from the mother loop. Luckily, in this case I already extracted this so I am able to just call the method variable (of the sub-method that contains the nested loop) according to what loop the index presents.  As an example, imagine, you control the loop with index 'i' by comparing it to 'j' or 'k'. I don't consider that very readable. The code within this module used to have many places with lots of nesting (and therefore unnecessary huge complexity). By refactoring (extracting sub-methods) I perhaps made the loop index renaming superfluous, which initially was or could have been very helpful.

Reply:
Of course, I agree, naming a loop index is a matter for the programmer's judgement. Ninety times out of a hundered, `i` is ideal; and nine times out the remaining ten, `j` is a fine index for a contained loop. But there's always that one other occasion.

(And of course nested loops are much less common now, what with all our functional programming idioms and our tendency to write shorter functions.)